src/app/common/stores/applications/SsvWeb/Validator.store.ts

```diff
@@ -496,14 +515,14 @@ class ValidatorStore extends BaseStore {
         totalCost = ssvStore.prepareSsvAmountToTransfer(walletStore.toWei(liquidationCollateralCost.add(networkCost).add(operatorsCost).toString()));
       }
       try {
-        const payLoad = [
-          this.keySharePublicKey,
-          this.keySharePayload?.operatorIds.map(Number).sort((a: number, b: number) => a - b),
-          this.keySharePayload?.sharesData,
-          `${totalCost}`,
-          await clusterStore.getClusterData(clusterStore.getClusterHash(this.keySharePayload?.operatorIds.sort())),
-        ];
-        resolve(payLoad);
+        const payload = {
+          keyStorePublicKey: this.keySharePublicKey,
+          operatorIds: this.keySharePayload?.operatorIds.map(Number).sort((a: number, b: number) => a - b),
+          sharesData: this.keySharePayload?.sharesData,
+          totalCost: `${totalCost}`,
+          clusterData: await clusterStore.getClusterData(clusterStore.getClusterHash(this.keySharePayload?.operatorIds.sort())),
```

Comment:
Please use ordered dict instead, and it's better to build some kind of method to re-use it in both places

https://chat.openai.com/share/952f594b-24af-48d4-92ec-e06b3b3fbbff

Reply:
done